python/tvm_ffi/module.py

```diff
@@ -170,6 +180,93 @@ def get_function(self, name: str, query_imports: bool = False) -> core.Function:
             raise AttributeError(f"Module has no function '{name}'")
         return func
 
+    def get_function_metadata(
```

Comment:
we can keep this function since metadata is not as common, make sure it aligns with function in global one